### DISCUSSION TITLE
[vcpkg/scripts] deprecate vcpkg_fail_port_install

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#17816] [Ideas]
- too restrictive
- might only apply to default triplets
- might block community triplets (e.g. mingw triplets)
- was added before the introduction of the supports field and the ci.baseline

**Comment by strega-nil** [comment]
Sounds good to me! If you like you can open a PR :)

**Comment by autoantwort** [comment]
When the supports field becomes a warning/error for the user this can definitive by done

**Reply by dg0yt** [reply]
But only if there is also a `supports` field for features.
